hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -8,6 +8,7 @@ define([
   'job-contract/app'
 ], function (_, moment, contractMock) {
   'use strict';
+  var controller;
```

Comment:
put this in the first `describe`, where you can see there are all the other "global" vars

Reply:
Done.